templates/CRM/Core/Form/RecurringEntity.tpl

```diff
@@ -111,19 +111,26 @@
     }
     $('#repetition_frequency_unit', $form).each(changeFrequencyUnit).change(changeFrequencyUnit);
 
-    function disableUnselected() {
-      $('input:radio[name=ends], input[name=repeats_by]', $form).not(':checked').siblings(':input').prop('disabled', true).removeClass('required');
-    }
-    disableUnselected();
+    function disableEnds() {
+      $("#repeat_absolute_date").prop('disabled', true);
+      $("#repeat_absolute_date").removeClass('required');
+      $('#start_action_offset').prop('disabled', true);
+      $("#start_action_offset").removeClass('required');
```

Comment:
This could be a one-liner in jQuery:

    $("#repeat_absolute_date, #start_action_offset").prop('disabled', true).removeClass('required');